app/src/main/java/com/jerboa/ui/components/post/PostListing.kt

```diff
@@ -307,7 +303,9 @@ fun PostName(
 
     Text(
         text = postView.post.name,
-        style = MaterialTheme.typography.titleLarge,
+        style = TextStyle(
+            fontSize = 22.sp,
```

Comment:
The titles are no longer controllable by the font size selection in settings.

Reply:
Hmm, not immediately sure how to handle this. Can I somehow edit the default font size of `titleLarge`, and is it safe to do so? Should I make a separate constant?

Reply:
multiply by it?

Reply:
Just use the existing material sizes, otherwise custom sizing won't work correctly. Try titleMedium.

Reply:
Especially when viewing image posts with titleMedium, the post title could use some extra visibility. I tweaked titleLarge for this purpose.